### PR TITLE
[Search Results] Add pagination

### PR DESCRIPTION
**Pull request by di-shu**
Spent: 30m
Estimate: 30m
Left: 0

### CODE REVIEW

**Review by Dmytro1991ua — COMMENTED** [review]
LGTM

File: src/service/api.tsx [hunk 1]
```diff
@@ -133,8 +133,8 @@ const api = {
   },
 
   async getSearchList(query: string, page: number = 1) {
-    const obj = await axios.get(`search/movie?${API_KEY}&query=${query}&page=${page}`);
-    return obj.data.results;
+    let obj = await axios.get(`search/movie?${API_KEY}&query=${query}&page=${page}`);
```
Comment (oleksandrstarov) [inline src/service/api.tsx:136]: Why let?

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]
Check build status and fix comment

**Review by oleksandrstarov — APPROVED** [review]
LGTM

**Comment by AnnaBolshak** [comment]
Could you check please, if add some movie to favourites and goto favourites page - there is the error in this branch 
image

**Pull request merged**